op-node/rollup/derive/l1_block_info.go

```diff
@@ -453,6 +453,19 @@ func L1InfoDeposit(rollupCfg *rollup.Config, sysCfg eth.SystemConfig, seqNumber
 	if isEcotoneButNotFirstBlock(rollupCfg, l2Timestamp) {
 		isIsthmusActivated := isIsthmusButNotFirstBlock(rollupCfg, l2Timestamp)
 		l1BlockInfo.BlobBaseFee = block.BlobBaseFee()
+
+		// Apply Cancun blob base fee calculation if this chain needs the L1 Pectra
+		// blob schedule fix (mostly Holesky and Sepolia OP-Stack chains).
+		if t := rollupCfg.PectraBlobScheduleTime; t != nil && block.Time() < *t {
```

Comment:
Do we aim to activate it based on L1 block timestamp, or L2 block timestamp? (I think this code is correct if we aim to use the L1 origin timestamp, which I think is better, but I don't can't find clarification of that choice).

Reply:
Yes it is intended to activate by L1 origin timestamp. Clarified in the fork field godoc and PR description.